packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -243,30 +245,28 @@ export default function MainFeedLayout({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [sortingEnabled, selectedAlgo, loadedSettings, loadedAlgo]);
 
-  const isMobile = useViewSize(ViewSize.MobileL);
-  const isLaptop = useViewSize(ViewSize.Laptop);
-
-  const getImage = () => {
-    if (isMobile) {
-      return cloudinary.feed.bg.mobile;
+  const getPadding = () => {
+    if (isV1Search) {
+      return '!pt-2';
     }
 
-    return isLaptop ? cloudinary.feed.bg.laptop : cloudinary.feed.bg.tablet;
+    return shouldUseFeedLayoutV1 && '!pt-0';
   };
 
   const FeedPageComponent = shouldUseFeedLayoutV1 ? FeedPageLayoutV1 : FeedPage;
 
   return (
-    <FeedPageComponent className="relative">
-      {searchVersion === SearchExperiment.V1 && !isFinder && (
-        <img
-          className="absolute left-0 top-0 w-full max-w-[58.75rem]"
-          src={getImage()}
-          alt="Gradient background"
+    <FeedPageComponent className={classNames('relative', getPadding())}>
+      {isV1Search && !shouldUseFeedLayoutV1 && !isFinder && <FeedGradientBg />}
```

Comment:
The evaluations become a bit tricky to maintain I feel like, let's see if we conclude experiments if it becomes easier. (no action needed)

Reply:
Yeah, hopefully, we get past at least one of the experiments to make it all easier.